content/en/methods/notifications.md

```diff
@@ -394,4 +394,4 @@ Invalid or missing Authorization header.
 
 {{< page-relref ref="methods/push" caption="push API methods" >}}
 
-{{< caption-link url="https://github.com/mastodon/mastodon/blob/main/app/controllers/api/v1/notifications_controller.rb" caption="app/controllers/api/v1/notifications_controller.rb" >}}
\ No newline at end of file
+{{< caption-link url="https://github.com/mastodon/mastodon/blob/main/app/controllers/api/v1/notifications_controller.rb" caption="app/controllers/api/v1/notifications_controller.rb" >}}
```

Comment:
Not sure why this change is here -- I edited the file through the Github web interface. This change can probably be deleted.